docs/source/userdoc/requirements/interns/adna-njuhovic/GPIO_requirements/informational_requirements.rst

```diff
@@ -99,12 +99,12 @@ Informational Requirements
    :priority: Low
    :submitted-by: Adna Njuhovic
    :modified-by:
-   :category: Functional
+   :category: Informational
    :safety-asil: 
    :references: 
    :verification-and-validation: 
 
-   GPIO pins in analog mode must support continuous signal processing for ADC (Analog to Digital) or DAC (Digital to Analog) operations.
+GPIO pins in analog mode must support continuous signal processing for ADC (Analog to Digital) or DAC (Digital to Analog) operations.
```

Comment:
nisi me razumjela sto se tice ovih info rqs; mi pisemo requiremente za firmware za GPIO, tj pisemo rqs kojima cemo reci developeru sta treba uraditi;
GPIO pins must support alternate functions for peripherals such as UART, SPI, or I2C.  u ovom rq ono sto nas zanima je samo funkcija koja ce omoguciti postavljanje af za ovaj pin